src/main/resources/db/migration/scriptFile.scala

```diff
@@ -0,0 +1 @@
+Tag docker image with HEAD commit
```

Comment:
How does this witchcraft work? Is it the same tagging we do in `build.sbt` but through Flyway?

Reply:
ahha ;) this is nothing, should not have commited this file